### PR TITLE
Bump teslajsonpy to 2.4.1

### PR DESCRIPTION
**Pull request by shred86**
Adds grid and home (load) power sensors for energy sites (solar systems).

### CODE REVIEW

**Review by alandtse — CHANGES_REQUESTED** [review]
Thanks. Can you also make sure to update pyproject.toml and do a poetry update instead of modifying poetry.lock directly?

**Comment by shred86** [comment]
Sorry about that - clearly have no idea what I'm doing with poetry. 😆 

So I just realized the only reason the solar sensor is being recreated is because I renamed it from `solar panel` to `solar power` to align the naming convention, `solar power`, `grid power`, `load power`. Instead of creating a breaking change, I can simply just retain the `solar panel` name so the `unique_id` in Home Assistant doesn't change...

**Comment by shred86** [comment]
Just made a [pull request](https://github.com/zabuldon/teslajsonpy/pull/336) over on teslajsonpy to revert the name. Sorry for the hassle and I should have caught this earlier but this will prevent the breaking change.

**Comment by alandtse** [comment]
Thanks for the patience. Just returned from vacation. 2.4.1 is pushed with the reversion.

**Comment by shred86** [comment]
Welcome back! Just updated the PR to include version 2.4.1 for teslajsonpy. I also updated the version of this integration to 2.4.0 in the manifest.json which is what I'm assuming it will be.

**Comment by alandtse** [comment]
You need to run `poetry update`. It should auto update dependencies and find 2.4.1.

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "tesla"
-version = "2.3.1"
```
Comment (alandtse) [inline pyproject.toml:3]: Don't bump the component version. That's done by CI.

**Review by alandtse — CHANGES_REQUESTED** [review]

File: custom_components/tesla_custom/manifest.json [hunk 2]
```diff
@@ -22,5 +22,5 @@
     }
   ],
   "iot_class": "cloud_polling",
-  "version": "2.3.1"
```
Comment (alandtse) [inline custom_components/tesla_custom/manifest.json:25]: Don't change this.

File: pyproject.toml (hunk 1 above)
Comment (shred86) [reply pyproject.toml]: Updated! Sorry about that.

File: custom_components/tesla_custom/manifest.json (hunk 2 above)
Comment (shred86) [reply custom_components/tesla_custom/manifest.json]: Updated! Sorry about that.

**Comment by shred86** [comment]
Sorry it wasn't updating to 2.4.1 when doing `poetry update`. It finally did after it was published for 13 minutes... guess I was just being impatient. 😄

**Review by alandtse — APPROVED** [review]

**Pull request merged**